[2025-06-22T15:20:02.877+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-22T15:20:02.897+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_hdfs manual__2025-06-22T08:20:00.466867+00:00 [queued]>
[2025-06-22T15:20:02.902+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_hdfs manual__2025-06-22T08:20:00.466867+00:00 [queued]>
[2025-06-22T15:20:02.902+0700] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-06-22T15:20:02.927+0700] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): start_hdfs> on 2025-06-22 08:20:00.466867+00:00
[2025-06-22T15:20:02.933+0700] {standard_task_runner.py:63} INFO - Started process 37016 to run task
[2025-06-22T15:20:02.938+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'start_kafka_hadoop_and_run_notebooks', 'start_hdfs', 'manual__2025-06-22T08:20:00.466867+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/start_DAG.py', '--cfg-path', '/tmp/tmpl5vtkjp9']
[2025-06-22T15:20:02.939+0700] {standard_task_runner.py:91} INFO - Job 6: Subtask start_hdfs
[2025-06-22T15:20:03.001+0700] {task_command.py:426} INFO - Running <TaskInstance: start_kafka_hadoop_and_run_notebooks.start_hdfs manual__2025-06-22T08:20:00.466867+00:00 [running]> on host Panda
[2025-06-22T15:20:03.086+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group_03' AIRFLOW_CTX_DAG_ID='start_kafka_hadoop_and_run_notebooks' AIRFLOW_CTX_TASK_ID='start_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T08:20:00.466867+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-22T08:20:00.466867+00:00'
[2025-06-22T15:20:03.087+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-22T15:20:03.087+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-22T15:20:03.088+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "'start-dfs.sh'"]
[2025-06-22T15:20:03.103+0700] {subprocess.py:86} INFO - Output:
[2025-06-22T15:20:03.958+0700] {subprocess.py:93} INFO - Starting namenodes on [localhost]
[2025-06-22T15:20:06.467+0700] {subprocess.py:93} INFO - Starting datanodes
[2025-06-22T15:20:09.849+0700] {subprocess.py:93} INFO - Starting secondary namenodes [Panda]
[2025-06-22T15:20:13.808+0700] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-22T15:20:13.809+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-22T15:20:13.826+0700] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=start_kafka_hadoop_and_run_notebooks, task_id=start_hdfs, execution_date=20250622T082000, start_date=20250622T082002, end_date=20250622T082013
[2025-06-22T15:20:13.856+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-22T15:20:13.872+0700] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-22T15:20:13.879+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
